mne/io/tests/test_reference.py

```diff
@@ -473,4 +474,41 @@ def test_add_reference():
     pytest.raises(ValueError, add_reference_channels, raw, 1)
 
 
+def test_bipolar_combinations():
+    """Test bipolar channel generation."""
+    channels = ['CH' + str(ni + 1) for ni in range(10)]
+    info = create_info(
+        ch_names=channels, sfreq=1000., ch_types=['eeg'] * len(channels))
+    raw_data = np.random.randn(len(channels), 1000)
+    raw = RawArray(raw_data, info)
+
+    def _check_bipolar(raw_test, ch_a, ch_b):
+        assert_array_equal(
+            raw_test.get_data(
+                picks=[raw_test.ch_names.index(ch_a + '-' + ch_b)])[0, :],
+            raw_data[channels.index(ch_a), :] -
+            raw_data[channels.index(ch_b), :])
```

Comment:
I find this a bit hard to read, maybe unpacking it would be a bit nicer.

```
picks = ...
get_data_res = raw.get_data(pick=...)
manual_a = raw_data[channels.index(...
manual_b = ...
assert_array_equal(get_data_res, manual_a - manual_b)
```

or so